extensions/azurePublish/src/node/luisAndQnA.ts

```diff
@@ -10,7 +10,7 @@ import { ILuisConfig, FileInfo, IBotProject } from '@botframework-composer/types
 import { AzurePublishErrors } from './utils/errorHandler';
 import { BotProjectDeployLoggerType } from './types';
 
-const botPath = (projPath: string) => path.join(projPath); // there should not be a need to specify the sub directory
+const botPath = (projPath: string) => path.join(projPath, 'ComposerDialogs'); // there should not be a need to specify the sub directory
```

Comment:
yes, this will break the new runtime. 

we will need to figure out a way to do this conditionally.

Reply:
why not add a guard on the new creation feature flag here?

Reply:
that would work.

Reply:
After discussing with @benbrown this morning, we will use the runtime key to decide between using the old vs new `generated` path.  

This is because there's an edge case using feature flags - a user could potentially be publishing a bot with old runtime even with new creation flag on (or vice versa).